src/dataServices/dynamoDbHelper.ts

```diff
@@ -22,7 +22,13 @@ export default class DynamoDbHelper {
         id: string,
         projectionExpression?: string,
     ): Promise<ItemList> {
-        const params = DynamoDbParamBuilder.buildGetResourcesQueryParam(id, resourceType, 2, projectionExpression);
+        const maxNumberOfVersions = 2;

```

Comment:
whoops I meant add it as a method param
```suggestion
maxNumberVersionsToGet: number,
projectionExpression?: string,
    ): Promise<ItemList> {

```

Reply:
Updated